docs/03-tutorials/00-application-connectivity/ac-03-register-manage-services.md

```diff
@@ -2,275 +2,117 @@
-                "clientId": "{CLIENT_ID}",
-                "clientSecret": "{CLIENT_SECRET}",
-                "requestParameters": {
-                    "headers": {
-                        "{CUSTOM_HEADER_NAME}": ["{CUSTOM_HEADER_VALUE}"]
-                    },
-                    "queryParameters":  {
-                        "{CUSTOM_QUERY_PARAMETER_NAME}": ["{CUSTOM_QUERY_PARAMETER_VALUE}"]
-                    }
-                }               
-            }
-        }  
-    }
-```
-
-## Use custom headers and query parameters for fetching API specification from URL
-
-You can specify additional headers and query parameters to inject to requests made to the specification URL.
-
->**NOTE:** These headers and query parameters are used only for requests for fetching an API specification and are not stored in the system.
-
-To register an API with a specification URL that requires specific custom headers and query parameters, add the **specificationRequestParameters.headers** and **specificationRequestParameters.queryParameters** objects to the **api** section of the service registration request body.
-
-```json
-    "api": {
-        "targetUrl": "https://sampleapi.targeturl/v1",
-        "specificationUrl": "https://sampleapi.spec/v1",
-        "specificationRequestParameters": {
-            "headers": {
-                "{CUSTOM_HEADER_NAME}": ["{CUSTOM_HEADER_VALUE}"]
-            },
-            "queryParameters": {
-                "{CUSTOM_QUERY_PARAMETER_NAME}" : ["{CUSTOM_QUERY_PARAMETER_VALUE}"]
-            }
-        },
-        "credentials": {
-            "basic": {
-                "username": "{USERNAME}",
-                "password": "{PASSWORD}"
-            }
-        }
-    }
-```
-
-## Register an OData API
-
-If the **api.spec** or **api.specificationUrl** parameters are not specified and the **api.type** parameter is set to `OData`, Application Registry tries to fetch the specification from the target URL with the `$metadata` path.
-
-For example, for the service with the following API, Application Registry will try to fetch the API specification from `https://services.odata.org/OData/OData.svc/$metadata`.
-
-```json
-"api": {
-  "targetUrl": "https://services.odata.org/OData/OData.svc",
-  "apiType": "OData",
-  "credentials": {
-    "basic": {
-      "username": "{USERNAME}",
-      "password": "{PASSWORD}"
-    }
-  }
-}
-```
\ No newline at end of file
+    > **CAUTION:** Mind that the `SERVICE_DISPLAY_NAME` in the **GATEWAY_URL** path must be in its [normalized form](./ac-04-register-secured-api.md#register-a-secured-api)! This means that, for example, if you used `test-basic-auth` as the service **displayName**, you're good to go, but if you used `"Test Basic Auth"`, you must replace it with `test-basic-auth` in the path. 
+
+2. Export the name of the test Pod as an environment variable:
+
+    ```bash
+    export POD_NAME=test-app-gateway
```

Comment:
```suggestion
    export POD_NAME={TEST_POD_NAME}
```

Reply:
Again, this is supposed to be a copy-pastey tutorial, so we're leaving an example name here on purpose. 
If we didn't use an exported variable, we'd use the placeholders :)